openedx/core/djangoapps/credit/tests/fake_update_min_grade.py

```diff
@@ -0,0 +1,70 @@
+"""
+Fake page for updating the minimum grade course requirement status for a user
+"""
+from django.contrib.auth.decorators import login_required
+from django.core.context_processors import csrf
+from django.utils.decorators import method_decorator
+from django.views.generic.base import View
+
+from openedx.core.djangoapps.credit.models import CreditCourse, CreditRequirement, CreditRequirementStatus
```

Comment:
move openedx, edxmako and xmodule imports to same block and opaque_keys to another block.